aries_cloudagent/protocols/didexchange/v1_0/tests/test_manager.py

```diff
@@ -236,6 +233,30 @@ async def test_create_request_implicit(self):
 
             assert conn_rec
 
+    # async def test_create_request_implicit_use_public_did(self):
```

Comment:
I'd need some help with this test. I want to setup a wallet with a public DID and assert that the did in the connection record is equal to the public DID, but I'm not sure how to setup/mock a wallet with a public DID

Reply:
Hi @domwoe sorry for the delay in responding to this.

Take a look at the tests for the in memory wallet, there is an example of creating a public did:

https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/wallet/tests/test_in_memory_wallet.py#L339

Reply:
Thx! This was actually pretty simple. Don't know why I couldn't get it working back then...